BeerDispancer/BeerDispencer.Domain/Entity/Dispencer.cs

```diff
@@ -0,0 +1,79 @@
+﻿using BeerDispencer.Domain.Abstractions;
+using BeerDispencer.Shared;
+
+namespace BeerDispencer.Domain.Entity
+{
+    public class Dispencer
```

Comment:
nit: typo -> Dispenser

Reply:
rename all occurencies of thisnshitty word (hope all)